packages/pipeline-services/src/validation/index.test.ts

```diff
@@ -23,7 +23,7 @@ describe("validate", () => {
     expect(problems).toHaveLength(0);
   });
 
-  it("should return an empty array if there are no problems", () => {
+  it("should find missing components", () => {
```

Comment:
Doesn't this remove the no errors test case, should we have both?

Reply:
we have other tests that cover no errors, this one was a bit redundant